README.md

```diff
@@ -35,6 +35,8 @@ Following are some of the key features of *eProsima DDS Router*:
   networks.
 * **Topic allowlisting**: it is possible to configure a *DDS Router* to forward just the user data belonging to a
   topic specified by the user.
+* **Dynamic topic discovery**: underlying entities required for communication under a specific topic are created only
```

Comment:
I would no say so.
The important part of dynamic module is that user does not need to know topics and types beforehand, not to minimize resources (which is not true)